crates/ruma-signatures/src/functions.rs

```diff
@@ -829,7 +835,28 @@ fn servers_to_check_signatures(
                 return Err(JsonError::field_missing_from_object("event_id"));
             }
         },
-        _ => (),
+        RoomVersionId::V3 => (),
+        RoomVersionId::V4 => (),
+        RoomVersionId::V5 => (),
+        RoomVersionId::V6 => (),
+        RoomVersionId::V7 => (),
```

Comment:
Please collapse these match arms into one and change the `()` to `{}` (much more common way to write an empty match arm).